about.html

```diff
@@ -0,0 +1,122 @@
+          <span
+            class="nav-icon material-icons menu" id="menu"
+          >
+            menu
+          </span>
+        </nav>
+        <div class="logo">
+          <img src="./Img/Svg/sony-logo-3066.png" alt="" />
+        </div>
+        <nav class="desktop-version" id="show">
+          <ul class="nav-bar">
+            <span
+              class="visible close" id="close"
+            >
+              <img src="./Img/Cl/close_FILL0_wght400_GRAD0_opsz48.png" alt="" />
+            </span>
+            <li> <a class="navbar-fun" href="./index.html">Home</a> </li>
+            <li> <a class="navbar-fun" href="#Home">Program</a> </li>
+            <li> <a class="navbar-fun" href="#Showcase">Pass Showcase</a></li>
+            <li> <a class="navbar-fun" href="#Sponsor">Sponsor</a> </li>
+            <li> <a class="navbar-fun" href="">Watch Live</a></li>
+          </ul>
+        </nav>
+      </header>
+      <section class="Home-1" id="Home">
+        <div class="Home-Back">
+          <h2 class="Home-1-h2">"Hello! Playstation world"</h2>
+          <h1>
+            PLAYSTATION & Insomniac Game<br />
+            GAMES SHOWCASE<br />
+            2022
+          </h1>
+          <span>
+            <p>
+              This year’s PLAYSTATION & Insomniac Game Showcase will present over 42
+              minutes of news and trailers for upcoming Playstation games. The event
+              will discuss everything Playstation owners could look forward to in 2021-2022
+              and beyond, with updates on the latest projects from Playstation Game
+              Studios and Insomniac Softworks. Play new games on day one. Plus,
+              enjoy hundreds of high-quality games with friends on console, PC,
+              or cloud. With games added all the time, there’s always something
+              new to play.
+            </p>
+          </span>
+          <p>
+            Please contact us via Email for any further questions about Playstation &
+            Insomniac Studio Showcase 2022!
+          </p>
+          <a href="#">help@ukplaystation.com </a>
+        </div>
+      </section>
+      <section class="main-page" id="Showcase">
+        <h1 class="main-page-1">See the past Showcases</h1>
+        <div class="border"></div>
+        <p>
+          Take a look at the last two Playstation & Insomniac Studio Games Showcase which took
+          place in 2020 and 2021.
+        </p>
+        <div class="card">
+          <div class="card-1">
+            <div class="card-1-m">
+              <!-- <img src="./Img/Image-required/forza4.jpg" alt=""> -->
+              <h2>2020</h2>
+              <p>Playstation & Insomniac Studio Games Showcase</p>
+            </div>
+          </div>
+          <div class="card-2">
+            <div class="card-1-m">
+              <h2>2021</h2>
+              <p>Playstation & Insomniac Studio Games Showcase</p>
+            </div>
+          </div>
+        </div>
```

Comment:
As requested by the previous reviewer please fix the mobile section as it currently has overflow, Try to decrease the font-size for your main title and remove fixed width and use flex or grid for a more responsive layout currently your layout is not responsive on  mobile screen please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/203848193-3f229cf5-31d4-48cd-ae4d-5f97738d99a6.png)